src/platform_impl/linux/x11/mod.rs

```diff
@@ -5,10 +5,10 @@
     target_os = "netbsd",
     target_os = "openbsd"
 ))]
+#![allow(non_fmt_panic)]
```

Comment:
I'm assuming you added this just as a temporary fix. I don't think it should be merged.

Reply:
Including this in a commit was completely accidental.